conf/messages.cy

```diff
@@ -546,11 +546,11 @@ youCannotAppeal.p3.lpp = Gall gymryd hyd at 5 diwrnod i’r taliad glirio ac ymd
 agent.youCannotAppeal.p3.lpp = Gall gymryd hyd at 5 diwrnod i’r taliad glirio ac ymddangos ar ei hanes taliadau. Os yw eisoes wedi talu, cadwch lygad ar yr hanes taliadau i weld pryd y mae’r taliad yn clirio.
 youCannotAppeal.p3.lsp = Gall gymryd hyd at 24 awr cyn i’r Ffurflen TAW ymddangos ar eich cyfrif TAW. Os ydych eisoes wedi’i chyflwyno, cadwch lygad ar eich cyfrif i weld pryd y mae’n ymddangos yn eich hanes cyflwyno.
 agent.youCannotAppeal.p3.lsp = Gall gymryd hyd at 24 awr cyn i’r Ffurflen TAW ymddangos ar y cyfrif TAW. Os yw eisoes wedi’i chyflwyno, cadwch lygad ar y cyfrif i weld pryd y mae’n ymddangos yn yr hanes cyflwyno.
-youCannotAppeal.p4 = If you need to cancel the VAT registration, you can do it through your VAT account.
-agent.youCannotAppeal.p4 = If you need to cancel the VAT registration, you can do it through your client‘s VAT details.
-youCannotAppeal.checkWhatYouOwe = Check and pay what you owe
-youCannotAppeal.returnToVATAccount = Return to your VAT account
-agent.youCannotAppeal.returnToVATAccount = Return to your client‘s VAT details
+youCannotAppeal.p4 = Os oes angen i chi ganslo’r cofrestriad TAW, gallwch wneud hyn drwy’ch cyfrif TAW.
+agent.youCannotAppeal.p4 = Os oes angen i chi ganslo’r cofrestriad TAW, gallwch wneud hyn drwy gyfrif TAW eich cleient.
```

Comment:
I think these need screenshots, the screenshots you've got currently miss the apostrophe off the word 'ganslo’r' or the apostrophe is wrong (hard to tell)